### ISSUE TITLE
1.2.36 breaks everything

### ISSUE THREAD

**Issue by ohitstom** [spicetify/spicetify-cli#2974]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
Spotify for Windows (64 bit)
1.2.36.955.gd8cc9372
Spicetify v2.36.5
xpui_2024-04-19_1713551583143_d8cc937
cef_122.1.13+gde5b724+chromium-122.0.6261.130
Runtime: Chrome
```

### 📝 Description

issues:
- having any custom app enabled causes a blank screen
- dropdown menus no longer work
- all context menus no longer work
- broken icons
- now playing bar css is unmapped
- main-rootlist-topSentinel has a bunch of unmapped children
- majority of left sidebar is unmapped

### 📸 Screenshots

issues i cant describe (not theme related btw):
![image](https://github.com/spicetify/spicetify-cli/assets/22730962/f50dbb37-40d9-4c62-b5da-b9f02f2e4953)
![image](https://github.com/spicetify/spicetify-cli/assets/22730962/6344e30d-5588-4dad-9523-c344a49bfd9d)

**Comment by rxri** [comment]
Everything except the mapped now playing bar css should be fixed via https://github.com/spicetify/spicetify-cli/pull/2973

**Comment by ohitstom** [comment]
> Everything except the mapped now playing bar css should be fixed via #2973

sweet - lots of the css map is broken in random areas sadge

**Comment by rxri** [comment]
Ok but then only list css map issue if there's already a fix on the way. Check if there's active pull request before making an issue about 'everything'

**Comment by ohitstom** [comment]
> Ok but then only list css map issue if there's already a fix on the way. Check if there's active pull request before making an issue about 'everything'

yes but alot of the css map is unfixed in that pr and there isnt mention of the custom app thing :)

**Comment by rxri** [comment]
> > Ok but then only list css map issue if there's already a fix on the way. Check if there's active pull request before making an issue about 'everything'
> 
> yes but alot of the css map is unfixed in that pr and there isnt mention of the custom app thing :)

Custom apps are broken because of `IconComponent` that I fixed lol

**Comment by ohitstom** [comment]
> Custom apps are broken because of `IconComponent` that I fixed lol

i see, my apologies

**Comment by rxri** [comment]
I will map now playing bar and left sidebar but `main-rootlist-topSentinel` didn't have anything mapped either. It uses auto-generated classes instead from `_getStyledClassName` which I'm not really proud of since they're sometimes very long but whatever

**Comment by ohitstom** [comment]
> I will map now playing bar and left sidebar but `main-rootlist-topSentinel` didn't have anything mapped. It uses auto-generated classes instead from `_getStyledClassName` which I'm not really proud of since they're sometimes very long but whatever

![image](https://github.com/spicetify/spicetify-cli/assets/22730962/ff3e8c0e-ecd8-482d-839b-3a52eaa1d4b1)
i could have sworn these were mapped in the previous spotify version

**Comment by rxri** [comment]
> > I will map now playing bar and left sidebar but `main-rootlist-topSentinel` didn't have anything mapped. It uses auto-generated classes instead from `_getStyledClassName` which I'm not really proud of since they're sometimes very long but whatever
> 
> ![image](https://private-user-images.githubusercontent.com/22730962/324293227-ff3e8c0e-ecd8-482d-839b-3a52eaa1d4b1.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM3OTY1NTEsIm5iZiI6MTcxMzc5NjI1MSwicGF0aCI6Ii8yMjczMDk2Mi8zMjQyOTMyMjctZmYzZThjMGUtZWNkOC00ODJkLTgzOWItM2E1MmVhYTFkNGIxLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDIyVDE0MzA1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWQ1NmNjMTNiOGVjZjVjNmFjOTQwMzdiMzE5MDhkMDA1MTJmMTFiYWE3MzkzYzFkZWVhZjhlYzJlZGQyNWUyZTQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.o5-bFKiaUEn3Igf6nml4rEojeR3-tEAFjZcXVlIMPWU) i could have sworn these were mapped in the previous spotify version i believe IjYxRc5luMiDPhKhZVUH UpiE7J6vPrJIa59qxts4 was .main-trackList-indexable or .main-trackList-trackListRowGrid

Then it wasn't mapped for a very long time. Feel free to create pull request mapping those

**Comment by ohitstom** [comment]
> Then it wasn't mapped for a very long time. Feel free to create pull request mapping those

it was mapped in the previous version, this one update broke comfy which is how im aware
